### ISSUE TITLE
Webmin not correct save iptabls rules with 2 ipset arg

### ISSUE THREAD

**Issue by a7lan** [webmin/webmin#1430]
Cant add Rules like this
`-A PREROUTING -p tcp -m tcp --dport 443 -m set --match-set usr src -m set ! --match-set ignore dst -j REDIRECT --to-ports 3129`

If I first add them to the iptables.save file, and then resave it through Webmin, then the rule looks like this
`-A PREROUTING -p tcp -m tcp -m set --dport 443 -j REDIRECT --to-ports 3129 --match-set usr src ! --match-set ignore dst`
I also get an error 
`iptables-restore v1.6.0: --match-set can be specified only once Error occurred at line: 60 Try iptables-restore -h or iptables-restore --help for more information."`

**Comment by jcameron** [comment]
Yeah, we don't yet support args for which the order matters. You'd need to split this into two rules.

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
